nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/DepthwiseConv2D.java

```diff
@@ -77,7 +77,20 @@ public DepthwiseConv2D(@NonNull INDArray input, @NonNull INDArray weights, INDAr
         this(wrapFilterNull(input, weights, bias), wrapOrNull(output), config);
     }
 
-    public DepthwiseConv2D() {
+
+
+    public DepthwiseConv2D(INDArray layerInput, INDArray depthWeights, Conv2DConfig conv2DConfig) {
+        this(wrapFilterNull(layerInput, depthWeights), null, conv2DConfig);
+    }
+
+    public DepthwiseConv2D(INDArray layerInput, INDArray depthWeights, INDArray bias, Conv2DConfig conv2DConfig) {
+        this(wrapFilterNull(layerInput, depthWeights, bias), null, conv2DConfig);
+
+
+    }
```

Comment:
layerInput, depthWeights and Conv2D config are not optional -> NonNull annotation
Bias is optional, however.
Avoid the unnecessary newlines (line 88/89) here too.